### ISSUE TITLE
Error is prodcued if zero ids are returne from query and reauthentication is required

### ISSUE THREAD

**Issue by ammaraziz** [Wytamma/GISAIDR#51]
If the query returns zero ids, `gisaidr::query` errors:

```
query(
  credentials = credentials,
  lineage =  "JN.1.1.2",
  location = 'Australia',
  fast = TRUE
  )
```
Output:
```
Selecting all 0 accession_ids.
Error in read.table(file = file, header = header, sep = sep, quote = quote,  : 
  no lines available in input
```

I have a script that queries many lineages in a for loop. If one lineage hits an error it stops the loop. Current solution, which might be the best solution is:

```
gisaid_login = function() {
    username = ""
    password = ""
    credentials <<- login(username = username, password = password)

}

lineages = list()
for (lineage in lineage_of_interest) {
    possibleError = tryCatch(
        {tmp = query(
            credentials = credentials,
            lineage = lineage,
            location = 'Australia',
            fast = TRUE
        )},
        error=function(e) e
    )
    if(inherits(possibleError, "error")) {
        gisaid_login()
        next
        }
    
    lineages = append(lineages, tmp)
}
```

Another odd side effect is that queries that are zero force reauthenticate:
```
> query(
+     credentials = credentials,
+     lineage =  "JN.1.1.2",
+     location = 'Australia',
+     fast = TRUE
+ )
Selecting all 0 accession_ids.
Error in read.table(file = file, header = header, sep = sep, quote = quote,  : 
  no lines available in input
> query(
+     credentials = credentials,
+     lineage =  "JN.1.1.2",
+     location = 'Australia',
+     fast = TRUE
+ )
Selecting all 0 accession_ids.
Error in parseResponse(response) : Error! Please login again.
```
Very strange. Could it be another anti-bioinformatician measures?

Posting the solution if others run into this problem.

**Comment by Wytamma** [comment]
Thanks @ammaraziz! This is related to the `fast = TRUE` query. When `get_accession_ids` fails (tries to read empty CSV of ids) the function exits and back cmd isn't sent (hence needing to log in again).

**Comment by Wytamma** [comment]
Solved in [v0.9.10](https://github.com/Wytamma/GISAIDR/releases/tag/v0.9.10)
